spec/lib/reporting/proofing_rate_report_spec.rb

```diff
@@ -65,21 +65,47 @@
   end
 
   describe '#reports' do
-    it 'calls IdentityVerificationReport correctly' do
-      expect(Reporting::IdentityVerificationReport).to receive(:new).with(
-        issuers: nil,
-        time_range: (start_date - 30.days)..start_date,
-      ).and_call_original
-      expect(Reporting::IdentityVerificationReport).to receive(:new).with(
+    before do
+      stub_const('Reporting::CloudwatchClient::DEFAULT_WAIT_DURATION', 0)
+
+      query_id = SecureRandom.hex
+
+      Aws.config[:cloudwatchlogs] = {
+        stub_responses: {
+          start_query: { query_id: query_id },
+          get_query_results: {
+            status: 'Complete',
+            results: [],
+          },
+        },
+      }
```

Comment:
we could also stub out `#data` or `#cloudwatch_client` on any of the instances we make? unsure if this is clearer

Reply:
Yeah, it’s probably six of one, half-dozen of the other - I think this is clear enough. 👍